pom.xml

```diff
@@ -25,6 +25,7 @@
     <slf4j.version>1.7.7</slf4j.version>
     <netty.version>4.1.29.Final</netty.version>
```

Comment:
Looks like this line isn't needed anymore. And dependencies with groupId `<groupId>io.netty</groupId>`  are also not needed (lines 129 - 165).